attribute/multivalueattribute.go

```diff
@@ -0,0 +1,88 @@
+package attribute
+
+import (
+	"log"
+
+	"github.com/getyoti/yoti-go-sdk/anchor"
+	"github.com/getyoti/yoti-go-sdk/yotiprotoattr"
+	"github.com/golang/protobuf/proto"
+)
+
+// MultiValueAttribute is a Yoti attribute which returns a multi-valued attribute
+type MultiValueAttribute struct {
+	*yotiprotoattr.Attribute
+	items   []*Item
+	anchors []*anchor.Anchor
+}
+
+// NewMultiValue creates a new MultiValue attribute
+func NewMultiValue(a *yotiprotoattr.Attribute) (*MultiValueAttribute, error) {
+	var attributeItems []*Item = ParseMultiValue(a.Value)
+
+	return &MultiValueAttribute{
+		Attribute: &yotiprotoattr.Attribute{
+			Name:        a.Name,
+			ContentType: a.ContentType,
+		},
+		items:   attributeItems,
+		anchors: anchor.ParseAnchors(a.Anchors),
+	}, nil
+}
+
+// ParseMultiValue recursively unmarshals and converts Multi Value bytes into a slice of Items
+func ParseMultiValue(data []byte) []*Item {
+	var attributeItems []*Item
+	protoMultiValueStruct := unmarshallMultiValue(data)
+
+	for _, multiValueItem := range protoMultiValueStruct.Values {
+		if multiValueItem.ContentType == yotiprotoattr.ContentType_MULTI_VALUE {
+			var parsedInnerMultiValueItems []*Item = ParseMultiValue(multiValueItem.Data)
+			var parsedInnerMultiValueItemsSingleItem = &Item{
+				contentType: yotiprotoattr.ContentType_MULTI_VALUE,
+				value:       parsedInnerMultiValueItems,
+			}
+			attributeItems = append(attributeItems, parsedInnerMultiValueItemsSingleItem)
```

Comment:
If both `parsedInnerMultiValueItemsSingleItem` and `value` are of type `Item`, you can bring this line and line 50 outside the if else.